Common.mk

```diff
@@ -135,6 +135,10 @@ BINARY_TARGETS?=$(call BINARY_TARGETS_FROM_FILES_PLATFORMS, $(BINARY_TARGET_FILE
 BINARY_TARGET_FILES?=
 SOURCE_PATTERNS?=.
 
+#################### HELM ######################
```

Comment:
You should be able to just set HAS_HELM_CHART in the cert manager makefile instead of introducing these new vars.

Reply:
Not sure how I missed that. Done.